controller_names:
  - right_arm_controller
  - left_arm_controller

right_arm_controller:
  action_ns: follow_joint_trajectory
  type: FollowJointTrajectory
  default: true
  joints:
    - RARM_JOINT0
    - RARM_JOINT1
    - RARM_JOINT2
    - RARM_JOINT3
    - RARM_JOINT4
    - RARM_JOINT5

left_arm_controller:
  action_ns: follow_joint_trajectory
  type: FollowJointTrajectory
  default: true
  joints:
    - LARM_JOINT0
    - LARM_JOINT1
    - LARM_JOINT2
    - LARM_JOINT3
    - LARM_JOINT4
    - LARM_JOINT5